src/EnergyPlus/MixedAir.cc

```diff
@@ -1338,6 +1338,7 @@ namespace MixedAir {
         static int ADUNum(0);
         int jZone;
         int i;
+        int j;
```

Comment:
No big deal but `j` could be defined down in the loop below where it is used instead of up here.